.evergreen/run-atlas-tests.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# variables
+PROJECT_DIRECTORY=${PROJECT_DIRECTORY:-$PWD}
+SWIFT_VERSION=${SWIFT_VERSION:-4.2}
+INSTALL_DIR="${PROJECT_DIRECTORY}/opt"
+OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+EXTRA_FLAGS="-Xlinker -rpath -Xlinker ${INSTALL_DIR}/lib"
+
+# enable swiftenv
+export SWIFTENV_ROOT="${INSTALL_DIR}/swiftenv"
+export PATH="${SWIFTENV_ROOT}/bin:$PATH"
+eval "$(swiftenv init -)"
+
+export PKG_CONFIG_PATH="${INSTALL_DIR}/lib/pkgconfig"
+
+# override where we look for libmongoc
+export LD_LIBRARY_PATH="${INSTALL_DIR}/lib"
+export DYLD_LIBRARY_PATH="${INSTALL_DIR}/lib"
+export DEVELOPER_DIR=/Applications/Xcode10.1.app
+
+swiftenv local $SWIFT_VERSION
+
+# run the tests
+ATLAS_REPL="$ATLAS_REPL" ATLAS_SHRD="$ATLAS_SHRD" ATLAS_FREE="$ATLAS_FREE" ATLAS_TLS11="$ATLAS_TLS11" ATLAS_TLS12="$ATLAS_TLS12" swift run AtlasConnectivity $EXTRA_FLAGS
```

Comment:
Does `swift run` work on targets too, not just executable products? Also, will we be actually running this / populating these environment variables from a private evg project?

Reply:
yup, `swift run AtlasConnectivity` runs this. I've omitted this from the products to prevent someone from doing `import AtlasConnectivity`. 
the output of `swift run --help` is confusing in that it says "Build and run an executable product" when in reality it builds and runs an executable target.
I've just set it up on our normal evg project, and all of these are private project variables added to the project settings via the UI. 
that said, from discussion in the comments on [DRIVERS-382](https://jira.mongodb.org/browse/DRIVERS-382) it sounds like Bernie wants us to use separate projects, though from a quick look at other drivers on evg it seems Node, Java, Python, maybe more all are just using their normal projects. thoughts @mbroadst ?

Reply:
yeah I think no need to set up a separate project, as long as you make sure never to `set -x` on the atlas shell script (which would leak the credentials)